docs/bigquery/snippets.py

```diff
@@ -1365,6 +1365,126 @@ def test_load_table_from_uri_truncate(client, to_delete):
     # [END bigquery_load_table_gcs_parquet_truncate]
 
 
+def test_load_table_add_column(client, to_delete):
+    dataset_id = 'load_table_add_column_{}'.format(_millis())
+    dataset_ref = client.dataset(dataset_id)
+    dataset = bigquery.Dataset(dataset_ref)
+    dataset.location = 'US'
+    dataset = client.create_dataset(dataset)
+    to_delete.append(dataset)
+
+    snippets_dir = os.path.abspath(os.path.dirname(__file__))
+    filepath = os.path.join(
+        snippets_dir, '..', '..', 'bigquery', 'tests', 'data', 'people.csv')
+
+    # [START bigquery_add_column_load_append]
+    # from google.cloud import bigquery
+    # client = bigquery.Client()
+    # dataset_ref = client.dataset('my_dataset')
+    # filepath = 'path/to/your_file.csv'
+
+    table_ref = dataset_ref.table('my_table')
```

Comment:
Variations of this table setup is present in all the samples, but isn't core to the snippet.  We should either clearly comment that this is setup, or move the table instantiation outside the snippet range and change the example to more clearly highlight the schema adaptation going on.

One possibility: walk the old/new schema and highlight how many fields were relaxed, or how many new fields were added with the print statements or asserts?